### ISSUE TITLE
Copying Blockchain "read only"

### ISSUE THREAD

**Issue by Jameshill89** [raspiblitz/raspiblitz#472]
Have tried downloading blockchain via torrent last week, once completed pi had to re-index chain so after 4 days and 2% of completed re-indexing I aborted. I decided to download bitcoin blockchain onto laptop to transfer it across. It transfer some data across then testnet and blocks etc are all "read only"

scp: /mnt/hdd/bitcoin/chainstate/LOCK: Read-only file system
scp: /mnt/hdd/bitcoin/chainstate/MANIFEST-043011: Read-only file system
./indexes: No such file or directory
scp: /mnt/hdd/bitcoin/testnet3: Read-only file system
scp: /mnt/hdd/bitcoin/blocks: Read-only file system

**Comment by istie** [comment]
Maybe it's this problem?
(My HDD case was faulty as well. after switching it the read-only problem was gone.)

> At first, I thought it was the drive's fault, so I swapped old drive to a new case. Surprisingly it stopped working after that. Finally, I swapped the cases and now it's working flawlessly.
> Sorry for wasting so much of your time @fluidvoice and I really appreciate your help.
> 
> TL;DR: The problem was with faulty HDD case
> _Originally posted by @szche in https://github.com/rootzoll/raspiblitz/issues/233#issuecomment-450739268_

**Comment by nyxnor** [comment]
please close this.
